mathics/builtin/arithmetic.py

```diff
@@ -1477,10 +1536,14 @@ class Gamma(_MPMathMultiFunction):
     #> N[Gamma[24/10], 100]
      = 1.242169344504305404913070252268300492431517240992022966055507541481863694148882652446155342679460339
     #> N[N[Gamma[24/10],100]/N[Gamma[14/10],100],100]
-     = 1.4
+     = 1.400000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
     #> % // Precision
      = 100.
 
+    #> Gamma[1.*^20]
+     : Overflow occurred in computation.
+     = Overflow[]
```

Comment:
In mathematica these should all overflow. The reason for the different behaviour is that mpmath has unbounded exponents.